config.js

```diff
@@ -152,6 +158,7 @@ export default {
   // supports RegExp or string.
   excludedCSSValues: [
     0,
+    '0px',
```

Comment:
huh curious where this is showing up - we have a lint rule in place that should prevent people from using units with 0 🤔

Reply:
@hannajones quite a few places see: https://searchfox.org/mozilla-central/search?q=%3A+0px%3B&path=*.css&case=false&regexp=false